projects/opendr_ws/src/ros_bridge/src/opendr_bridge/bridge.py

```diff
@@ -107,3 +107,51 @@ def from_ros_pose(self, ros_pose):
         pose = Pose(data, confidence)
         pose.id = pose_id
         return pose
+
+    def to_ros_boxes(self, box_list):
```

Comment:
Please also add documentation for these two function to [rosbridge.md](https://github.com/tasostefas/opendr_internal/blob/master/docs/reference/rosbridge.md)